examples/apps/generic_client/README.md

```diff
@@ -123,3 +124,10 @@ With Docker:
     --address "0x9Be28B132aeE1b2c5A1C50529a636cEd807842cd" --mode "listing" \
     --workload_id "echo-result" --in_data "Hello"
 ```
+
+### Echo workload with input data as plain text
```

Comment:
I was not sure what "plain text" means. It could mean not in JSON format for example.  Please add something like this:

`### Echo workload with input data as unencrypted plain text`